assets/src/scss/new-identity/style.scss

```diff
@@ -121,13 +124,15 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+a:where(.standalone-link, .comment-reply-link, #cancel-comment-reply-link) {
+  color: var(--link--color) !important;
```

Comment:
I know this is redundant but we'll need to enforce to use `--link--color`. For example the `#cancel-comment-reply-link` uses a grey color.

![Screenshot 2023-04-21 at 08 05 04](https://user-images.githubusercontent.com/77975803/233620930-749d2a14-f168-4cac-81b3-b00f2b184b04.png)

Reply:
Maybe it's wanted though, I mean maybe the "Cancel reply" link should remain the same grey as it is now? We should check with the design team

Reply:
There's a similar link style in the cookies box for the "Reject all" link:

<img width="416" alt="Screenshot 2023-04-21 at 13 37 09" src="https://user-images.githubusercontent.com/6949075/233626393-09ac0b7a-9ae1-4315-a793-59a9d3de17e8.png">

Reply:
I think that we don't need the `!important` anymore since the cancel link remains grey? 🤔